src/components/Loading/index.jsx

```diff
@@ -1,5 +1,15 @@
 import React from "react";
+import { Spin } from 'antd';
 
 export default function Loading() {
-  return <div>loading...</div>;
+  return <Spin
+    spinning={true}
+    style={{
+      width: '100%',
+      flex: 1,
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center'
```

Comment:
@SirM2z Shall we create a separate style for this?

Reply:
Do you mean to write into less file?

Reply:
Already updated